pkgs/html/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 0.15.7-wip
 
 - Require Dart `3.6`
+- Fix a bug in DOM parsing where `<br>` tags does not create a new line when html is converted to text.
```

Comment:
It's no longer a bug then, change the changelog entry to convey adding a new `textContent` method instead.